<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signup</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/6488645.png</normaloff>../icons/6488645.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 255, 255);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>89</y>
       <width>317</width>
       <height>425</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);border-radius: 15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="email">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255); border-radius: 10px;
color: rgb(154, 154, 154);
border: 1px solid gray;
  </string>
      </property>
      <property name="text">
       <string>   Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255); border-radius: 10px;
border: 1px solid gray;color: rgb(154, 154, 154);

   </string>
      </property>
      <property name="text">
       <string>   Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signup_button">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
font: 12pt &quot;Segoe UI Emoji&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 170, 255);
border: 1px solid gray

   </string>
      </property>
      <property name="text">
       <string>Signup</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 5, 0);
font: 28pt &quot;Segoe UI Emoji&quot;;
</string>
      </property>
      <property name="text">
       <string>Signup</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setecho">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>200</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icons/eye-closed-icon-512x459-ivg8wjcx.png</normaloff>../icons/eye-closed-icon-512x459-ivg8wjcx.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>320</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Already have an account?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="redirect_login">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>320</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 170, 255);</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255); border-radius: 10px;
color: rgb(154, 154, 154);
border: 1px solid gray;
  </string>
      </property>
      <property name="text">
       <string>   Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="warn">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>240</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Fill in all the blanks!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
